.github/workflows/ci.yaml

```diff
@@ -8,5 +8,4 @@ on:
 
 jobs:
   shared:
-    uses: pantos-io/ci-workflows/.github/workflows/python-ci.yml@84c18c1b5bf447e4a02f94c45c5266abab52bd55
-    secrets: inherit  
+    uses: pantos-io/ci-workflows/.github/workflows/python-ci.yml@v1.0.3
```

Comment:
Here we should still reference `v1` and not `v1.0.3`. Then, in the `ci-workflows` repo we should tag the commit of `v1.0.3` also with `v1`

According to https://github.com/actions/toolkit/blob/main/docs/action-versioning.md:
> Binding to a major version is the latest of that major version ( e.g. v1 == "1.*" )

Reply:
I don't like the idea of not setting specific versions specially when we're playing with the current code. The good thing is that we will not need new changes in `common` for new releases of the CI 🤔 I will change it for the sake of convenience. Thanks for the point.